docs/maintainer-notes.md

```diff
@@ -32,3 +32,35 @@ Or regenerate everything:
 ```cli
 ./generate --all
 ```
+
+## Creating a new client
+
+If you want to create a new AWS client, there are a few steps to take. Below
+is an example for DynamoDB.
+
+1. Clone the async-aws/aws repository.
+1. Run `composer install`
+1. Create `./src/DynamoDb/DynamoDbClient.php` and add an empty `DynamoDbClient` class.
+1. Update `./manifest.json`
+   1. Look at the official SDK for resources in `./src/data`. ([DynamoDB example](https://github.com/aws/aws-sdk-php/tree/3.133.23/src/data/dynamodb/2012-08-10))
+   1. Add links to `source`,  `documentation`, `example`, `pagination`, `waiter` etc.
+   1. Leave the `methods` key empty. (`"methods": {}`)
+1. Run `./generate DynamoDb` and generate the operations you need. Don't generate operations that you dont need.
+1. Complete the test stubs created in `./src/DynamoDb/Tests`
```

Comment:
+ `Use the [AWS Api Reference](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Operations.html) to fill the expected Input/Result`.

Reply:
Thank you